vcx/wrappers/ios/vcx/ConnectMeVcx.h

```diff
@@ -172,6 +172,8 @@ withSelectedCredentials:(NSString *)selectedCredentials
 withConnectionHandle:(vcx_connection_handle_t)connection_handle
     withCompletion:(void (^)(NSError *error))completion;
 
+- (int)vcxShutdown:(BOOL *)deletePool;
```

Comment:
The official vcx.h labels this is parameter as "delete_wallet" but everywhere here it's saying "deletePool".  The fact that it deletes the wallet is much more important than the fact that it deletes the pool.  I think it should be changed to "deleteWallet" in every location.

Reply:
ok, I will do that